docs/userGuide/developingASite.md

```diff
@@ -42,6 +42,7 @@ MarkBind will generate the site in a folder named `_site` in the current directo
 | `-s`, `--site-config <file>` | Specify the site config file (default: site.json) |
 | `--one-page <file>` | Render and serve only a single page from your website. |
 | --no-open | Don't open browser automatically. |
+| `--baseurl <override-base-url>`	| Build site with the baseUrl overridden. |

```

Comment:
Move it above `--one-page <file>`, let's try to make it in alphabetical order.

Reply:
Okay thanks, will do! 
Also...should I add the code formatting for `--no-open` or open another PR for that?

Reply:
> Also...should I add the code formatting for `--no-open` or open a new PR

Please do so in this PR, since it is quite trivial. Thanks!

Reply:
Please remove the tab after the `<override-base-url>`:

```
| `--baseurl <override-base-url>` | Build site with the baseUrl overridden. |
```